docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+services:
+  frontend:
+    build: frontend
+    ports:
+      - "4200:80"
+    networks:
+      - back-tier
+  
+  backend:
+    build: backend
+    ports:
+      - "8080:8080"
+    networks:
+      - back-tier
+
+networks:
+  back-tier: {}
```

Comment:
Skoro już nie ma dwóch sieci to przydałaby się ogólniejsza nazwa niż _back-tier_.

Reply:
Racja, zaraz idzie poprawka